crowdsourcing/serializers/requester.py

```diff
@@ -2,10 +2,26 @@
 
 from crowdsourcing import models
 from rest_framework import serializers
+import datetime
 
 class RequesterSerializer(serializers.ModelSerializer):
+    total_projects = serializers.SerializerMethodField()
+    current_projects = serializers.SerializerMethodField()
+    waiting_projects = serializers.SerializerMethodField()
+
```

Comment:
but then we will need to pass extra parameter to module serializer ! 
PS : inspired from Worker REST API